server/db/utils.go

```diff
@@ -25,10 +27,22 @@ func (s *Store) setupConnForMigrations(dsn string) (*sql.DB, error) {
 		return nil, fmt.Errorf("failed to ping DB: %w", err)
 	}
 
-	db.SetMaxIdleConns(*s.settings.MaxIdleConns)
-	db.SetMaxOpenConns(*s.settings.MaxOpenConns)
-	db.SetConnMaxLifetime(time.Duration(*s.settings.ConnMaxLifetimeMilliseconds) * time.Millisecond)
-	db.SetConnMaxIdleTime(time.Duration(*s.settings.ConnMaxIdleTimeMilliseconds) * time.Millisecond)
+	maxIdleConns := max(*s.settings.MaxIdleConns/10, 5)
+	maxOpenConns := max(*s.settings.MaxOpenConns/10, 10)
```

Comment:
May need to review these as we run performance tests. Keeping a conservative approach for now to avoid having the plugin potentially overload the db,